src/index.js

```diff
@@ -30,23 +20,15 @@ module.exports = (/* options */) => ({
   // Fix sourcemaps
   result(stylelintResult, filepath) {
     const lineCorrection = sourceMapsCorrections[filepath]
-    const newWarnings = stylelintResult.warnings.reduce((prevWarnings, warning) => {
-      if (ignoredRules.includes(warning.rule)) return prevWarnings
-      const correctedWarning = Object.assign(warning, {
+    const warnings = stylelintResult.warnings.map(warning =>
+      Object.assign(warning, {
         // Replace "brace" with "backtick" in warnings, e.g.
         // "Unexpected empty line before closing backtick" (instead of "brace")
```

Comment:
This wouldn't be correct if it you had a declaration block inside of your backticks, right? Maybe we should do some check and only do that replace if it's on line 0 or the last line of the block?